src/Microsoft.ML.CodeGenerator/Templates/AzureImageClassification/Model/NormalizeMapping.cs

```diff
@@ -0,0 +1,356 @@
+﻿// ------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version: 16.0.0.0
+//  
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+// ------------------------------------------------------------------------------
+namespace Microsoft.ML.CodeGenerator.Templates.AzureImageClassification.Model
+{
+    using System.Linq;
+    using System.Text;
+    using System.Collections.Generic;
+    using System;
+    
+    /// <summary>
+    /// Class to produce the template output
+    /// </summary>
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
+    internal partial class NormalizeMapping : NormalizeMappingBase
+    {
+        /// <summary>
+        /// Create the template output
+        /// </summary>
+        public virtual string TransformText()
+        {
+if(Target == CSharp.GenerateTarget.Cli){ 
+CLI_Annotation();
+ } else if(Target == CSharp.GenerateTarget.ModelBuilder){ 
+MB_Annotation();
+ } 
```

Comment:
If not difficult, we can cleanup the formatting of the generated file.

Reply:
## Resolved